biome.json

```diff
@@ -66,6 +69,10 @@
   "json": {
     "formatter": {
       "enabled": true
+    },
+    "parser": {
+      "allowComments": true,
+      "allowTrailingCommas": true
```

Comment:
I always thought comments and trailing commas were strictly forbidden in e.g. `package.json`s? Isn't this the difference between Json and Json-C?

Reply:
It is forbidden, but we have trailing commas and comments in tsconfig files.